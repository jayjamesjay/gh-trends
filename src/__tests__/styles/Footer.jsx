import React from 'react';
import { render } from '@testing-library/react';
import 'jest-styled-components';
import Footer from '../../styles/Footer';

test('renders default <Footer />', () => {
  const { asFragment } = render(<Footer />);
  expect(asFragment()).toMatchSnapshot();
});
