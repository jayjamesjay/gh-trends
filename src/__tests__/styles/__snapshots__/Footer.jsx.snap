// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default <Footer /> 1`] = `
<DocumentFragment>
  .c0 {
  margin: 1rem;
  padding: 0.5rem;
  text-align: center;
}

<footer
    class="c0"
  />
</DocumentFragment>
`;
